### DISCUSSION TITLE
Question.

### DISCUSSION THREAD

**Discussion by zahmbie1** [ScratchNitro/Scratch-Nitro#18] [Q&A] [answered]
The question is... why.

**Comment by JackyBorderCollie** [comment, marked as answer]
We want others to experience Scratch Nitro features too

**Comment by zahmbie1** [comment]
Why not contribute to ScratchAddons like somebody would do?

**Comment by Marc92020** [comment]
> contribute to ScratchAddons like somebody would do

Well I contribute to Scratch Addons but this is a cool idea.

**Comment by JackyBorderCollie** [comment]
> > contribute to ScratchAddons like somebody would do
> 
> Well I contribute to Scratch Addons but this is a cool idea.

How to contribute to Scratch Addons?

**Comment by Marc92020** [comment]
> > > contribute to ScratchAddons like somebody would do
> > 
> > 
> > Well I contribute to Scratch Addons but this is a cool idea.
> 
> How to contribute to Scratch Addons?

Find their repository and make an issue. Also there is an issue where you can add yourself but you have to contribute or else you might be removed from the repository
